chapter04/c04_family_call.rpy

```diff
@@ -3,7 +3,7 @@ label c04_family_call:
     c "Um... [Pause] H-How... How is your family doing?"
     k "!"
     Kenta's body tensed up at the word "family", as if ice had been thrown down his back.
-    Kenta's hazel eyes narrowed at me, looking somewhat fearful and threatening.
```

Comment:
Thanks for the explanation! I'll skip the spoilers haha, I'm enjoying things so far. This is why editors always have to read through everything first but as a proofreader I can relax and read things as they come.